src/main/java/io/vertx/starter/WebVerticle.java

```diff
@@ -52,6 +52,8 @@
   public WebVerticle() {
     try {
 
+
```

Comment:
Can you please revert the blank lines change?

Reply:
👷 Done